src/frontend/src/components/mainWindow.ts

```diff
@@ -0,0 +1,53 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { icLogo } from "./icons";
+import { footer } from "./footer";
+
+export type mainWindowProps = {
```

Comment:
Just a thought: for readability (and to avoid going back and forth between this type and the function definition) you could just write the type in the function definition; I don't think it needs to be exported. If you do decide to keep the type here please capitalize the first letter for consistency with other types!